components/test-suite/src/loci/tests/testng/Configuration.java

```diff
@@ -828,7 +828,7 @@ private Length getPhysicalSize(String valueKey, String unitKey) {
     String units = currentTable.get(unitKey);
     try {
       UnitsLength unit = units == null ? UnitsLength.MICROMETER : UnitsLength.fromString(units);
-      return physicalSize == null ? null : UnitsLength.create(new Double(physicalSize), unit);
+      return physicalSize == null ? null : UnitsLength.create(DataTools.parseDouble(physicalSize), unit);
```

Comment:
Excluding for now due to test failures in a number of formats. I have so far traced it down to this section of code as the cause.

Reply:
I believe 4662b65c55 fixes this, but leaving excluded until after 7.3.0 is out just in case.